src/forms/Slushbucket.tsx

```diff
@@ -0,0 +1,237 @@
+                this.resetSearchString();
+              }}
+            />
+          </div>
+        </div>
+
+        <div
+          style={{
+            display: "flex"
+          }}
+        >
+          <div
+            className={styles.Results}
+            style={{
+              marginRight: "20px"
+            }}
+          >
+            <div className={`${styles.SelectedRow}`}>
+              <b>Available</b>
+            </div>
+            <Scrollbars autoHeight autoHeightMin={400} autoHeightMax={400}>
+              {choices
+                .filter(choiceItem => {
+                  if (this.props.readonly) {
+                    return false;
+                  }
+                  if (this.state.searchString === "") {
+                    // if nothing is typed show all results
+                    return true;
+                  } else {
+                    // if user typed search string only show those that contain string
+                    // TODO sort by search string ?
+                    return choiceItem
+                      .display
+                      .toLowerCase()
+                      .includes(this.state.searchString.toLowerCase());
+                  }
+                })
+                .map((choiceItem, i) => {
+                  return (
+                    <div
+                      tabIndex={i + 1}
+                      key={choiceItem.value + '_' + i}
+                      className={`${styles.ResultRow} ${selected.includes(
+                        choiceItem.value
+                      )
+                        ? styles.Active
+                        : ""}`}
+                      onMouseDown={() => {
+                        if (
+                          selected.filter(e => e === choiceItem.value).length === 0
+                        ) {
+                          selected.push(choiceItem.value);
+                          updateModelValue(selected);
+                        } else {
+                          updateModelValue(
+                            selected.filter(e => e !== choiceItem.value)
+                          );
+                        }
+                      }}
+                    >
+                      {choiceItem.display}
+                    </div>
+                  );
+                })}
+            </Scrollbars>
+          </div>
+          <div
+            className={styles.Results}
+            style={{
+              marginLeft: "20px"
+            }}
+          >
+            <div className={`${styles.SelectedRow}`}>
+              <b>Selected</b>
+            </div>
+            <Scrollbars autoHeight autoHeightMin={400} autoHeightMax={400}>
+              {selected
+                .map((selectedItem) => {
+                  // lookup complete { value, display} object in choices array
+                  return choices.filter(choice => choice.value === selectedItem)[0];
+                })
+                .sort((nameA, nameB) => {
```

Comment:
I don't like sorting in the input component. I think both keeping them in the order that they were selected in or in the same order as they are in choices are fine.

Reply:
Keeping them in order they were selected is easiest to implement and it is also easy for users that just selected some new organisations. This will be then always on bottom of list.

I just checked here:
https://demo.lizard.net/management/#/data_management/rasters/730d6675-35dd-4a35-aa9b-bfb8155f9ca7
that selected organisations are already in alphabetic order from the backend so there is no need so sort. I remove the sort.